### ISSUE TITLE
[ysql] YB successfully starts despite YSQL webserver failing to bind

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#5621]
Jira Link: [DB-1988](https://yugabyte.atlassian.net/browse/DB-1988)
Use netcat to listen on 1300 to make webserver bind fail: `nc -l 13000`
Start a local yb cluster with `yb-ctl create`

```
I0909 00:01:50.413642  8507 webserver.cc:150] Webserver listen spec is 127.0.0.1:13000
I0909 00:01:50.413931  8507 webserver.cc:326] Webserver: set_ports_option: cannot bind to 127.0.0.1:13000: 98 (Address already in use)
W0909 00:01:50.414037  8507 net_util.cc:499] Failed to bind to 0.0.0.0:13000. Trying to use lsof to find any processes listening on the same port:
I0909 00:01:50.414065  8507 net_util.cc:502] $ export PATH=$PATH:/usr/sbin ; lsof -n -i 'TCP:13000' -sTCP:LISTEN ; for pid in $(lsof -F p -n -i 'TCP:13000' -sTCP:LISTEN | cut -f 2 -dp) ; do  while [ $pid -gt 1 ] ; do    ps h -fp $pid ;    pid=$(sed 's/.* (.*) [^ ] \([0-9]*\).*/\1/g' /proc/$pid/stat);  done ; done
2020-09-09 00:01:50.425 UTC [8483] LOG:  database system is ready to accept connections
W0909 00:01:50.579003  8507 net_util.cc:509] COMMAND  PID   USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
nc      8357 bogdan    3u  IPv6 7146374      0t0  TCP *:13000 (LISTEN)
nc      8357 bogdan    4u  IPv4 7146375      0t0  TCP *:13000 (LISTEN)
bogdan    8357  3084  0 00:01 pts/4    S+     0:00 nc -l 13000
bogdan    3084  3080  0 Jul08 pts/4    Ss     0:00 -bash
bogdan    3080     1  0 Jul08 ?        Ss     4:21 tmux new-session -s yugaware tmux source-file ~/.tmux.yugaware.session
F0909 00:01:50.579149  8507 ybc-internal.cc:30] Check failed: 'g_palloc_fn' Must be non NULL·
Fatal failure details written to /n/users/bogdan/yugabyte-data/node-1/disk-1/yb-data/tserver/logs/yb-tserver.FATAL.details.2020-09-09T00_01_50.pid8507.txt
F20200909 00:01:50 ../../src/yb/common/ybc-internal.cc:30] Check failed: 'g_palloc_fn' Must be non NULL·
    @     0x7f87879e70fc  yb::LogFatalHandlerSink::send()
    @     0x7f8783a51376  google::LogMessage::SendToLog()
    @     0x7f8783a4e7da  google::LogMessage::Flush()
    @     0x7f8783a518a9  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f8787392c04  yb::YBCPAlloc()
    @     0x7f8787390ff5  DupYBStatusMessage
    @           0xa0cfc5  HandleYBStatus
    @     0x7f877a5b4230  webserver_worker_main
    @           0x7be89d  StartBackgroundWorker
    @           0x7ce294  maybe_start_bgworkers
    @           0x7d0cec  PostmasterMain
    @           0x71cc5a  PostgresServerProcessMain
    @           0x71ce59  main
    @     0x7f87866ff825  __libc_start_main
    @           0x49a7c9  _start
    @              (nil)  (unknown)

*** Check failure stack trace: ***
    @     0x7f87879e54e1  yb::(anonymous namespace)::DumpStackTraceAndExit()
    @     0x7f8783a4ec8d  google::LogMessage::Fail()
    @     0x7f8783a50dfd  google::LogMessage::SendToLog()
    @     0x7f8783a4e7da  google::LogMessage::Flush()
    @     0x7f8783a518a9  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f8787392c04  yb::YBCPAlloc()
    @     0x7f8787390ff5  DupYBStatusMessage
    @           0xa0cfc5  HandleYBStatus
    @     0x7f877a5b4230  webserver_worker_main
    @           0x7be89d  StartBackgroundWorker
    @           0x7ce294  maybe_start_bgworkers
    @           0x7d0cec  PostmasterMain
    @           0x71cc5a  PostgresServerProcessMain
    @           0x71ce59  main
    @     0x7f87866ff825  __libc_start_main
    @           0x49a7c9  _start
    @              (nil)  (unknown)
2020-09-09 00:01:50.617 UTC [8483] LOG:  background worker "YSQL webserver" (PID 8507) was terminated by signal 6: Aborted
2020-09-09 00:01:50.617 UTC [8483] LOG:  terminating any other active server processes
2020-09-09 00:01:50.629 UTC [8483] LOG:  all server processes terminated; reinitializing
2020-09-09 00:01:50.659 UTC [8524] LOG:  database system was interrupted; last known up at 2020-09-09 00:01:50 UTC
2020-09-09 00:01:50.695 UTC [8524] LOG:  database system was not properly shut down; automatic recovery in progress
2020-09-09 00:01:50.700 UTC [8524] LOG:  invalid record length at 0/1000108: wanted 24, got 0
2020-09-09 00:01:50.700 UTC [8524] LOG:  redo is not required
2020-09-09 00:01:50.729 UTC [8483] LOG:  database system is ready to accept connections
```

However the processes are up fine:
> | Node 1: yb-tserver (pid 8396), yb-master (pid 8393)

**Comment by tedyu** [comment]
This seems to be dup of issue #5862 for which I have a patch ready.

**Comment by tedyu** [comment]
Marking this resolved since fix for #5862 has been integrated.
